cmd/minishift/cmd/config/config.go

```diff
@@ -100,17 +100,20 @@ var (
 	CacheImages  = createConfigSetting("cache-images", SetSlice, nil, nil, true)
 
 	// Pre-flight checks (before start)
-	SkipCheckKVMDriver    = createConfigSetting("skip-check-kvm-driver", SetBool, nil, nil, true)
-	WarnCheckKVMDriver    = createConfigSetting("warn-check-kvm-driver", SetBool, nil, nil, true)
-	SkipCheckXHyveDriver  = createConfigSetting("skip-check-xhyve-driver", SetBool, nil, nil, true)
-	WarnCheckXHyveDriver  = createConfigSetting("warn-check-xhyve-driver", SetBool, nil, nil, true)
-	SkipCheckHyperVDriver = createConfigSetting("skip-check-hyperv-driver", SetBool, nil, nil, true)
-	WarnCheckHyperVDriver = createConfigSetting("warn-check-hyperv-driver", SetBool, nil, nil, true)
-	SkipCheckIsoUrl       = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true)
-	WarnCheckIsoUrl       = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true)
-	SkipCheckVMDriver     = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true)
-	WarnCheckVMDriver     = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true)
-
+	SkipCheckKVMDriver      = createConfigSetting("skip-check-kvm-driver", SetBool, nil, nil, true)
+	WarnCheckKVMDriver      = createConfigSetting("warn-check-kvm-driver", SetBool, nil, nil, true)
+	SkipCheckXHyveDriver    = createConfigSetting("skip-check-xhyve-driver", SetBool, nil, nil, true)
+	WarnCheckXHyveDriver    = createConfigSetting("warn-check-xhyve-driver", SetBool, nil, nil, true)
+	SkipCheckHyperVDriver   = createConfigSetting("skip-check-hyperv-driver", SetBool, nil, nil, true)
+	WarnCheckHyperVDriver   = createConfigSetting("warn-check-hyperv-driver", SetBool, nil, nil, true)
+	SkipCheckIsoUrl         = createConfigSetting("skip-check-iso-url", SetBool, nil, nil, true)
+	WarnCheckIsoUrl         = createConfigSetting("warn-check-iso-url", SetBool, nil, nil, true)
+	SkipCheckVMDriver       = createConfigSetting("skip-check-vm-driver", SetBool, nil, nil, true)
+	WarnCheckVMDriver       = createConfigSetting("warn-check-vm-driver", SetBool, nil, nil, true)
+	SkipCheckVBoxInstalled  = createConfigSetting("skip-check-vbox-installed", SetBool, nil, nil, true)
+	WarnCheckVBoxInstalled  = createConfigSetting("warn-check-vbox-installed", SetBool, nil, nil, true)
```

Comment:
The names of the flag does not look consistent with other similar flag names. I think the purpose of `WarnCheckVBoxInstalle` is similar as `WarnCheckHyperVDriver`.

Reply:
Yes they are similar, theseis flags can be used to skip a preflight test, so i wanted to make it explicit that we are skipping the VBox installation check. hence named it so.